src/metabase/analytics/prometheus.clj

```diff
@@ -5,6 +5,7 @@
 
   Api is quite simple: [[setup!]] and [[shutdown!]]. After that you can retrieve metrics from
   http://localhost:<prometheus-server-port>/metrics."
+  (:refer-clojure :exclude [set!])
```

Comment:
I'm not sure if this actually does anything, given its a special form. Does no harm though, expresses the intention.